tests/test_patterns.py

```diff
@@ -0,0 +1,263 @@
+import pytest
+
+from roboragi.patterns import find_requests
+
+
+@pytest.mark.parametrize('given,expected', [
+    ('{K}', ['K']),
+    ('{Bakemonogatari}', ['Bakemonogatari']),
+    ('{Steins;Gate}', ['Steins;Gate']),
+    ('{Kill la Kill}', ['Kill la Kill']),
+    ('{ヒナまつり}', ['ヒナまつり']),
+    ('{Re:Zero} {No Game No Life}', ['Re:Zero', 'No Game No Life']),
+    ('', []),
+    ('{}', []),
+    ('{ }', []),
+    ('{  }', []),
+    ('{trailing_whitespace }', []),
+    ('{ leading_whitespace}', []),
+    ('{ wrapping_whitespace }', []),
+    ('no_whitespace{before}', []),
+    ('{{extended}}', []),
+    ('{{partial_before}', []),
+    ('{partial_after}}', []),
+    ('{nested{after}}', []),
+    ('{{nested}before}', []),
+    ('{parens_after}(', []),
```

Comment:
As per my previous comment, this test should probably be a valid match (I can't think of a good reason to call out "(" as being specially invalid). Can I recommend removing this one and adding two new tests in its place:

`('{Gunbuster}, {Diebuster}', ['Gunbuster', 'Diebuster']),
('{Nisekoi}: Words Go Here', ['Nisekoi']),`